common/primitives/constants.go

```diff
@@ -30,3 +25,8 @@ const (
 const (
 	ScheduleWorkflowIDPrefix = "temporal-sys-scheduler:"
 )
+
+var (
+	// DefaultWorkflowTaskTimeout sets the Default Workflow Task timeout for a Workflow
+	DefaultWorkflowTaskTimeout = 10 * time.Second * debugtimeout.Multiplier
+)
```

Comment:
This is not a const anymore, but it feels const-ish enough to leave it in this file.